server/src/metadata.rs

```diff
@@ -48,6 +48,26 @@ pub mod entities_metadata_query {
         entities: Vec<EntityMetadata>,
     }
 
+    #[derive(Serialize)]
+    enum CreatorType {
+        Person,
+        // Should we support this type too?
+        // Organization,
+    }
+
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    struct Creator {
```

Comment:
Put this type after struct EntityMetadata, see https://github.com/serlo/documentation/wiki/Best-Practices#order-source-files-from-abstract-to-concrete